cmd/plasmacli/subcmd/eth/root.go

```diff
@@ -40,6 +40,7 @@ func RootCmd() *cobra.Command {
 		FinalizeCmd(),
 		DepositCmd(),
 		WithdrawCmd(),
```

Comment:
Lets add a line break under `WithdrawCmd`. and have the status be on a new line.

Separates the cmds that are action based versus a health check